### ISSUE TITLE
[Proposal] Log format

### ISSUE THREAD

**Issue by itscaro** [kkdai/youtube#168]
Since a while I'm using a modified version which prints out like this, is it OK have this kind of info as output in --verbose (--debug) without HTTP debug and (--trace to have HTTP debug)

```
youtubedr download -q hd720 -c mp4 <videoID> <videoID> <videoID>
2021/04/11 01:14:14 download to directory .
check ffmpeg is installed....
2021/04/11 01:14:14 ▶ <videoID>: '<video Title>'
2021/04/11 01:14:14 Video (Quality '720p' | FPS '30' | Codec 'video/mp4; codecs="av01.0.05M.08"')
2021/04/11 01:14:14 Audio (Channels '2' | Codec 'audio/mp4; codecs="mp4a.40.2"')
2021/04/11 01:14:14 Downloading video file...
153.98 MiB / 271.77 MiB 57 % [==================================>---------------------------] 1m15s ] 4.29 MiB/s
```

master version as of 36071ffb6b127b7c5c6ecd916edac8012758da10
```
2021/04/21 10:18:53 GET https://youtube.com/get_video_info?video_id=<videoId>&eurl=https://youtube.googleapis.com/v/<videoId>
2021/04/21 10:18:53 download to directory .
2021/04/21 10:18:53 Video '<videoTitle>' - Quality '' - Codec 'audio/mp4; codecs="mp4a.40.2"'
2021/04/21 10:18:53 Download to file=<filename>.m4a
2021/04/21 10:18:53 GET https://youtube.com/embed/<videoId>?hl=en
2021/04/21 10:18:53 GET https://youtube.com/s/player/ae5b2092/player_ias.vflset/en_US/base.js
2021/04/21 10:18:53 GET https://r3---sn-cv0tb0xn-jqbk.googlevideo.com/videoplayback?expire=1619014733&ei=7d9_YISbObn4xN8Pwequ0Ag&ip=176.169.27.51&id=o-AOLuZC4wgUWXWWnqBxaVcIbWZOdx3noix-es06_h4mM-&itag=140&source=youtube&requiressl=yes&mh=Vm&mm=31%2C29&mn=sn-cv0tb0xn-jqbk%2Csn-25glen7y&ms=au%2Crdu&mv=m&mvi=3&pl=19&initcwndbps=1985000&vprv=1&mime=audio%2Fmp4&ns=Pd8UD_ZLfSWAgZUKGZAfFEgF&gir=yes&clen=2203581&dur=136.115&lmt=1598623915795960&mt=1618992540&fvip=3&keepalive=yes&fexp=24001373%2C24007246&c=WEB&txp=5511222&n=IQB_-w2PipIfinHB&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cdur%2Clmt&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgSl1dOt_cJG8XHYb3ilIBlLAmAONJqyYvEeqhX4OakwcCIQClT8UXT03dfE4uNJpD3-RpjqJavE1LpF5q37hEJ_3sQQ%3D%3D&sig=AOq0QJ8wRQIgWAeG365utOdoN5iHyVxiaVTIyRPkqtvjK9q2dtd35N0CIQCaoFCI6UOFZQD0y3Y22Pf7K6kYC2Jde62K3othd3Yvjw==
2.10 MiB / 2.10 MiB 100 % [==============================================================] 0s ] 63.98 MiB/s

--
# with quality

2021/04/21 14:44:58 Video '<videoTitle>' - Quality '1080p' - Codec 'video/mp4; codecs="av01.0.08M.08"'
```

**Comment by Julian-Chu** [comment]
hi @itscaro  could you please attach original version format as well,  then we could compare both easily.

**Comment by Julian-Chu** [comment]
hi @itscaro ,

your proposal looks great to me,  would you like to make a PR ?

**Comment by corny** [comment]
solved by 8eabf5fbeafd51d7aa6dd2f19816cd7dac46c205
